wandb/sdk/launch/queue_driver/standard_queue_driver.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from typing import Any, Dict, List, Optional
+
+from wandb.apis.internal import Api
+from wandb.sdk.launch.agent2.job_set import JobSet
+from wandb.sdk.launch.queue_driver.abstract import AbstractQueueDriver
+from wandb.sdk.launch.utils import PRIORITIZATION_MODE
+
+logger = logging.getLogger(__name__)
+
+
+class StandardQueueDriver(AbstractQueueDriver):
+    def __init__(self, api: Api, job_set: JobSet):
+        self.api = api
+        self.job_set = job_set
+
+    def pop_from_run_queue(self) -> Optional[Dict[str, Any]]:
+        # get highest prio job
+        if self.job_set.metadata.get(PRIORITIZATION_MODE) == "V0":
+            job = sorted(
+                self.job_set.jobs, key=lambda j: (j["priority"], j["createdAt"])
+            )[0]
+        else:
+            job = sorted(self.job_set.jobs, key=lambda j: j["createdAt"])[0]
+
+        # attempt to acquire lease
+        lease_result = self.job_set.lease_job(job["id"])
```

Comment:
`JobSet.lease_job` is async so it'll return a `Future`.  You would need to await it and then wait for `job_set` to be updated, then this can return the job.  That implies this method (and probably all of them) needs to be async so we don't block the event loop.

The workflow is:

1. Fire off `lease_job`
2. Wait for `job_set` to update (we don't have a good mechanism for doing this correctly right now.  ideally job_set has a subscription interface that can call back subscribers with updates instead of forcing them to poll)
3. Check that the selected job is, in fact, leased in the new `job_set` state
4. Return the job